code/game/turfs/turf.dm

```diff
@@ -3,7 +3,7 @@ GLOBAL_LIST_EMPTY(station_turfs)
 /// Any floor or wall. What makes up the station and the rest of the map.
 /turf
 	icon = 'icons/turf/floors.dmi'
-	vis_flags = VIS_INHERIT_ID|VIS_INHERIT_PLANE // Important for interaction with and visualization of openspace.
+	vis_flags = VIS_HIDE | VIS_INHERIT_ID | VIS_INHERIT_PLANE// Important for interaction with and visualization of openspace.
```

Comment:
Does this break anything else that uses turf in vis_contents?

Reply:
yeah i should look for that, last i checked mirage_holders still worked (somehow, i have no clue why)

Reply:
only other thing i could find was the gateway so i think i fixed that